wiki/_Sidebar.md

```diff
@@ -1,35 +1,32 @@
-## Usage
 * [Getting Started](./Getting-Started.md)
-* [Example systemd Setup](./Example-systemd-Setup.md)
-* [Important Software](./Important-Software.md)
+* [Configuration](./Configuration:-Overview.md)
+    * [Input](./Configuration:-Input.md)
+    * [Outputs](./Configuration:-Outputs.md)
+    * [Key Bindings](./Configuration:-Key-Bindings.md)
+    * [Switch Events](./Configuration:-Switch-Events.md)
+    * [Layout](./Configuration:-Layout.md)
+    * [Named Workspaces](./Configuration:-Named-Workspaces.md)
+    * [Miscellaneous](./Configuration:-Miscellaneous.md)
+    * [Window Rules](./Configuration:-Window-Rules.md)
+    * [Layer Rules](./Configuration:-Layer-Rules.md)
+    * [Animations](./Configuration:-Animations.md)
+    * [Gestures](./Configuration:-Gestures.md)
+    * [Debug Options](./Configuration:-Debug-Options.md)
 * [Floating Windows](./Floating-Windows.md)
 * [Tabs](./Tabs.md)
 * [Layer‐Shell Components](./Layer%E2%80%90Shell-Components.md)
-* [IPC, `niri msg`](./IPC.md)
-* [Application-Specific Issues](./Application-Issues.md)
-* [Xwayland](./Xwayland.md)
 * [Gestures](./Gestures.md)
+* Other software
+    * [Important Software](./Important-Software.md)
+    * [Example systemd Setup](./Example-systemd-Setup.md)
+    * [Xwayland](./Xwayland.md)
+    * [Application-Specific Issues](./Application-Issues.md)
+* [IPC, `niri msg`](./IPC.md)
 * [Packaging niri](./Packaging-niri.md)
 * [FAQ](./FAQ.md)
-
-## Configuration
-* [Overview](./Configuration:-Overview.md)
-* [Input](./Configuration:-Input.md)
-* [Outputs](./Configuration:-Outputs.md)
-* [Key Bindings](./Configuration:-Key-Bindings.md)
-* [Switch Events](./Configuration:-Switch-Events.md)
-* [Layout](./Configuration:-Layout.md)
-* [Named Workspaces](./Configuration:-Named-Workspaces.md)
-* [Miscellaneous](./Configuration:-Miscellaneous.md)
-* [Window Rules](./Configuration:-Window-Rules.md)
-* [Layer Rules](./Configuration:-Layer-Rules.md)
-* [Animations](./Configuration:-Animations.md)
-* [Gestures](./Configuration:-Gestures.md)
-* [Debug Options](./Configuration:-Debug-Options.md)
-
-## Development
-* [Design Principles](./Design-Principles.md)
-* [Developing niri](./Developing-niri.md)
-* [Fractional Layout](./Fractional-Layout.md)
-* [Redraw Loop](./Redraw-Loop.md)
-* [Animation Timing](./Animation-Timing.md)
+* Development
+    * [Design Principles](./Development:-Design-Principles.md)
+    * [Developing niri](./Development:-Developing-niri.md)
+    * [Fractional Layout](./Development:-Fractional-Layout.md)
+    * [Redraw Loop](./Development:-Redraw-Loop.md)
+    * [Animation Timing](./Development:-Animation-Timing.md)
```

Comment:
I kinda prefer the layout with the headers more?

Reply:
i prefer headers too, but my desire to bring configuration up near the top breaks up the content a little too much. if you don't care about the configuration-part being high up, i can revert this change.

Reply:
I don't think it's that important yes